chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -682,7 +712,20 @@ func (b *BitcoindNotifier) RegisterSpendNtfn(outpoint *wire.OutPoint,
 		return nil, err
 	}
 
-	if txOut == nil {
+	// If the output is unspent, then we'll write it to the cache with the
```

Comment:
This matters more for neutrino, but if the cached height hint is the _same_ as the current height, then we can skip the `GetUTXO` query all together. This will save us filter fetches which will (eventually) add to the cache, and also possibly fetch a block.

Reply:
We'd still need to fetch a block to manually check for the spend, no?

Reply:
At tip now, it should be cached so doesn't make a difference.